helm_deploy/laa-crown-court-proceeding/templates/ingress.yaml

```diff
@@ -34,16 +34,25 @@ spec:
     {{- end }}
   {{- end }}
   rules:
-    {{- if .Values.ingress.internalHost }}
-    - host: {{ $.Values.ingress.internalHost.host | quote }}
+    {{- range .Values.ingress.hosts }}
+    - host: {{ .host | quote }}
       http:
         paths:
-          - path: {{ $.Values.ingress.internalHost.path }}
+          {{- range .paths }}
+          - path: {{ . }}
             pathType: "Prefix"
             backend:
               service:
                 name: {{ $fullName }}
                 port:
                   number: {{ $svcPort }}
+          - path: {{ $.Values.actuator.health.path }}
+            pathType: "Exact"
+            backend:
+              service:
+                name: {{ $fullName }}
+                port:
+                  number: {{ $.Values.actuator.port }}
+          {{- end }}
```

Comment:
Not sure we need this if we're exposing everything. It looks like this was there to expose the health endpoint in prod where we were being more restrictive and only allowing swagger docs to be accessed.